app/models/services/UserServiceImpl.scala

```diff
@@ -60,6 +61,7 @@ class UserServiceImpl @Inject() (userDAO: UserDAO) extends UserService {
           lastName = profile.lastName,
           fullName = profile.fullName,
           email = profile.email,
+          role = Some("Student"),
```

Comment:
IMHO "Student" and other role-Strings should be extracted into app/utils/silhouette/Authorization.scala (something like ```val ROLE_STUDENT = "Student"```) and then be referenced instead of a direct String. This avoids typos etc.